src/api/goods.ts

```diff
@@ -10,10 +10,16 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      goods.sort((goodA, goodB) => goodA.name.localeCompare(goodB.name));
+
+      return goods.slice(0, 5);
+    });
```

Comment:
Chaining methods looks more clear, and we'll have only one return, that also looks better for me
`return getAll()
  .then(goods => gods.sort(......).slice(.....)` of course with formating )